src/sources/index.ts

```diff
@@ -0,0 +1,47 @@
+import { SignalKind } from "../signals";
+import getAppVersion from "./app-version";
+import getBrowserKind from "./browser-kind";
+import getDocumentAttributes from "./document-attributes";
+import getDocumentProperties from "./document-properties";
+import getErrorTrace from "./error-trace";
+import getEvalLength from "./eval-length";
+import getFunctionBind from "./function-bind";
+import getLanguages from "./languages";
+import getPermissions from "./permissions";
+import getPluginsArray from "./plugins-array";
+import getPluginsLength from "./plugins-length";
+import getProcess from "./process";
+import getRTT from "./rtt";
+import getUserAgent from "./user-agent";
+import getUserAgentData from "./user-agent-data";
+import { getWebDriver } from "./web-driver";
+import getWebGL from "./webgl";
+import getWindowExternal from "./window-external";
+import getWindowProperties from "./window-properties";
+import getWindowSize from "./window-size";
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export default function getSources() {
+  return {
+    [SignalKind.AppVersion]: getAppVersion,
```

Comment:
Minor: this enum made sense when the property names were meaningless "`s`+number" values. Now the enum only adds inconvenience. Since the detectors use the `ComponentDict` types with known properties, TypeScript does the job of checking for property names.

Reply:
I took a step further and renamed the sources folder to `components` as I think it fits pretty well.